desidiff/nb/desidiff.ipynb

```diff
@@ -167,9 +178,25 @@
     "            print(tid)\n",
     "            for b in difflux.keys():\n",
     "                w=numpy.where(difmask[b] ==0)[0]\n",
-    "                plt.plot(difwave[b][w],difflux[b][w],color='red')\n",
-    "                plt.plot(newwave[b][w],newflux[b][w],color='blue',alpha=0.5)\n",
+    "#                 plt.plot(difwave[b][w],difflux[b][w],color='red')\n",
```

Comment:
This commented out thing looks the same as the code above.  Is there a difference between them?

Reply:
There isn't a difference, but it was the code from the master that you had written and so I didn't want to delete it, since originally the outputs looked different with my code until you fixed the issue in coadd.